client/mksubvolume.cc

```diff
@@ -43,6 +43,7 @@ using namespace BtrfsUtils;
 
 string target;
 
+bool force = false;
```

Comment:
Where is force used?

Reply:
Oh, I'm sorry, this should have been used as part of the if condition in the exception handler above - will fix.

Reply:
Thinking about it: It _could_ also just be left out, so existing suvolumes are reused automatically. I'm not sure if this is a good idea though.